Production/python/get_mcm.py

```diff
@@ -47,6 +88,9 @@ def getA(query,debug):
         print jstr
         return None
 
+def parser_callback(option, opt, value, parser):
```

Comment:
You can get this from: https://github.com/kpedro88/CondorProduction/blob/master/python/parseConfig.py#L5

Reply:
Done